README.rst

```diff
@@ -10,7 +10,7 @@ About
 A native Kerberos client implementation for Python on Windows. This module
 mimics the API of `pykerberos <https://pypi.python.org/pypi/pykerberos>`_ to
 implement Kerberos authentication with Microsoft's Security Support Provider
-Interface (SSPI). It supports Python 2.7 and 3.4+.
+Interface (SSPI). It supports Python 3.6+.
```

Comment:
Let's not do this yet. Until PyMongo 3 goes EOL in March 2023 we should continue to support Python 2.7 and 3.5 in the modules we maintain that PyMongo depends on. Happily that doesn't really add any burden here.

Reply:
Done